tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -298,7 +299,9 @@ The ``initialize_example_module`` and ``uninitialize_example_module`` functions
 called respectively when Godot loads our plugin and when it unloads it. All
 we're doing here is parse through the functions in our bindings module to
 initialize them, but you might have to set up more things depending on your
-needs. We call the ``GDREGISTER_CLASS`` macro for each of our classes in our library.
+needs. We call the ``GDREGISTER_RUNTIME_CLASS`` macro for each of our classes
+in our library. This will make them run only in game, like the default for
+GDScript, but there are more choices in ``godot-cpp``.
```

Comment:
```suggestion
needs. We call the ``GDREGISTER_RUNTIME_CLASS`` macro for each of our classes
in our library. This will make them run only in game, like the default for
GDScript.
```

I think we can simplify that. Just saying there are more choices and then not supplying those choices would be a bad UX. This could be worded once we could redirect them to a separate page where this is actually explained.